pywbemtools/pywbemcli/_cmd_class.py

```diff
@@ -830,16 +831,22 @@ def cmd_class_tree(context, classname, options):
             classname = None
 
         else:
-            # get the subclass hierarchy either complete or starting at the
+            # Get the subclass hierarchy either complete or starting at the
             # optional CLASSNAME
             classes = context.conn.EnumerateClasses(
                 ClassName=classname,
                 namespace=options['namespace'],
                 DeepInheritance=True)
+
+        # Get correct case sensitive classname for target class
+        if classname:
```

Comment:
This logic causes the if not to be used if the class had superclasses (see `classname = None`, a few lines up). Is that the intention? If so, why?

Reply:
The god if indenting got me.  wanted to sent the correct case classname on to the formatter but only for subclass tree since by definition, a superclass tree always goes up to root, which explains setting classname to None.